Documentation/kubernetes-on-baremetal.md

```diff
@@ -0,0 +1,33 @@
+# Kubernetes Installation on Bare Metal &amp; CoreOS
+
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
+After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using he kubectl CLI tool.
+
+## Deployment Prerequisites
+
+### CoreOS Installation
+
+For all nodes running Kubernetes components (master & workers), you must use CoreOS version 773.1.0+ for the kubelet to be present in the image.
+
```

Comment:
Might be nice to put an intro paragraph about when to install to disk vs PXE. And maybe the difference between iPXE and PXE with a common use-case as an example.